### PR TITLE
Add flags to portfolio model

### PR DESCRIPTION
**Pull request by ajang06**
Flags MVP complete

### CODE REVIEW

**Comment by sophiagavrila** [comment]
@ajang06 Changes look great, but Merging is blocked due to insufficient testing (we need 80% coverage on the edited code).

Please defer to this[ SonarCloud report](https://sonarcloud.io/component_measures?id=1058-Sophia-Gavrila-Forge_Forge-Backend&pullRequest=28&metric=new_coverage&view=list) listing the classes that we need some JUnit tests for

**Review by MadeFaux — APPROVED** [review]

**Review by sophiagavrila — APPROVED** [review]
great work on 81% test coverage! ready to merge pending other reviewers' approval

**Pull request merged**
